patches/server/0013-Paper-Plugins.patch

```diff
@@ -949,10 +949,8 @@ index 0000000000000000000000000000000000000000..392eb260b44d5f9e685ce09596c19f0a
 +
 +    @Override
 +    public void close() throws IOException {
-+        try {
++        try (this.jar; this.libraryLoader) {
 +            super.close();
```

Comment:
Yes I am aware this looks odd, but doing a try-with-resources (which can also take local vars and fields as-is) will ensure that a) everything is closed, so I don't have to cascade 2 or 3 different `try-finally`s, and b) it also handles cases where IOExceptions are thrown, it doesn't discard anything by "appending" exceptions via `Throwable::addSuppressed`